drivers/scheduler/k8s/k8s.go

```diff
@@ -2439,6 +2600,49 @@ func (k *K8s) Destroy(ctx *scheduler.Context, opts map[string]bool) error {
 			}
 		}
 	}
+
+	for _, appSpec := range ctx.App.SpecList {
+		t := func() (interface{}, bool, error) {
```

Comment:
Can we consider giving a more meaningful name to the anonymous function?

Reply:
I can. But there's no point to it. It can be called `task`.
The code I used is consistent with the rest of the function. (you can check the surrounding code, it also uses `t`).
Also, the variable name is actually the name of the parameter (`t`) to which the variable is passed. I guess it would also make sense to change the name of the parameter too.

@mkoppal-px ?